packages/sil-governance/src/routes/dashboard/index.dom.spec.ts

```diff
@@ -88,7 +88,7 @@ describe("DOM > Feature > Dashboard", () => {
     let readToggle = longProposal.children[2].children[0].children[1];
 
     await click(readToggle);
-    let description = longProposal.children[2].children[0].children[0].textContent;
+    let description = longProposal.children[2].children[0].children[0].children[0].children[0].textContent;
```

Comment:
Isn't there an easier way to find the text? Something that is more maintainable? @albertandrejev?